tests/src/python/CMakeLists.txt

```diff
@@ -61,6 +61,8 @@ ADD_PYTHON_TEST(PyQgsVectorColorRamp test_qgsvectorcolorramp.py)
 ADD_PYTHON_TEST(PyQgsVectorFileWriter test_qgsvectorfilewriter.py)
 ADD_PYTHON_TEST(PyQgsVectorLayer test_qgsvectorlayer.py)
 ADD_PYTHON_TEST(PyQgsZonalStatistics test_qgszonalstatistics.py)
+ADD_PYTHON_TEST(PyQgsMapLayerRegistry test_qgsmalayerregistry.py)
```

Comment:
Looks like the typo here is what [failed your Travis CI builds](https://travis-ci.org/qgis/QGIS/jobs/94895415#L844-L846).